WooCommerce/Classes/Extensions/UIImage+Woo.swift

```diff
@@ -606,4 +612,8 @@ private extension UIImage {
     enum Metrics {
         static let defaultWooLogoSize = CGSize(width: 30, height: 18)
     }
+
+    enum Configurations {
+        static let barButtonItemSymbol = UIImage.SymbolConfiguration(pointSize: 22, weight: .regular, scale: .medium)
```

Comment:
nit: isn't better to not customize the size here, but just where is needed? In this way, the icon is not reusable in other points of the app, if needed.

Reply:
I think symbol images work a bit differently from pre-existing images from an asset or Gridicons library, since the image initialization requires `UIImage.SymbolConfiguration` and I haven't found an API to update its `pointSize`/`weight`/`scale` after initialization.

When we start replacing existing Gridicons usage in the future, we could consider passing a `UIImage.SymbolConfiguration` in `UIImage+Woo` with a static function instead and some default configurations are provided. For this PR, I thought I'd keep changes only to the affected navigation bar button items. What do you think?

Reply:
Probably the best way, If I understood correctly the problem, is to not use a `UIBarButtonItem`, but a UIButton that will be initialized with a specific size that respects the size that we want.

> For this PR, I thought I'd keep changes only to the affected navigation bar button items. What do you think?

Yes I agree, I don't see any other elements that are affected by this at the moment 👍

Reply:
> Probably the best way, If I understood correctly the problem, is to not use a UIBarButtonItem, but a UIButton that will be initialized with a specific size that respects the size that we want.

Did you mean using a `UIButton` inside a `UIBarButtonItem` so that the image is more customizable and reusable? It sounds worth trying when we are replacing more images with SF symbols  👍